microsoftgraph/docs-conceptual/authentication-commands.md

```diff
@@ -39,11 +39,29 @@ There are three ways to allow delegated access using `Connect-MgGraph`:
     Connect-MgGraph -AccessToken $AccessToken
     ```
 
+#### Use delegated access using own App Id
+
+Follow the steps below to create custom applications that you can use to connect to Microsoft Graph PowerShell. Use this approach if you need to limit the consent permissions granted for Graph PowerShell usage.
```

Comment:
A small change here to make it easier to understand.

Follow the steps below to create custom applications that you can use to connect to Microsoft Graph PowerShell. Use this approach if you need to isolate and limit the consent permissions granted for Graph PowerShell usage.